apps/myjobs/test/models/filesystem_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class FilesystemTest < ActiveSupport::TestCase
+  test "copy_dir doesn't allow command injection" do
+    o,e,s = Filesystem.new.copy_dir("/dev/null';echo INJECTED;'", '/dev/null')
+    refute_match /INJECTED/, o
```

Comment:
I get the same error  that travis does. I guess we need to rescue or expect the exception here?